integrations/tensorflow/bindings/python/pyiree/tf/support/tf_utils.py

```diff
@@ -302,7 +297,18 @@ def supports_cxx_serialization() -> bool:
     return True
 
 
-class _TfFunctionWrapper(object):
+def _normalize_numpy(result: np.ndarray):
+  """Normalizes TF/Lite's output to match IREE's"""
+  if np.isscalar(result):
+    # convert_to_tensor isn't reversible via .numpy()
```

Comment:
I don't find this comment very useful. I'd find one lik ethe one for np.bool more useful (how IREE and TFLite differ).